src/screens/UserPortal/UserScreen/UserScreen.tsx

```diff
@@ -7,7 +7,7 @@ import type { TargetsType } from 'state/reducers/routesReducer';
 import styles from './UserScreen.module.css';
 import { Button } from 'react-bootstrap';
 import UserSidebarOrg from 'components/UserPortal/UserSidebarOrg/UserSidebarOrg';
-import ProfileDropdown from 'components/ProfileDropdown/ProfileDropdown';
```

Comment:
Why it has been removed?

Reply:
@varshith257 because it was not of use anymore, ProfileDropdown was just taking an extra space